mkimg.lima.sh

```diff
@@ -24,6 +24,17 @@ profile_lima() {
             apks="$apks libseccomp runc containerd tini-static device-mapper-libs"
             apks="$apks docker-engine docker-openrc docker-cli docker"
         fi
+        if [ "${LIMA_INSTALL_PODMAN}" == "true" ]; then
+            apks="$apks libcap libseccomp yajl crun libffi libintl libblkid"
+            apks="$apks libmount pcre glib conmon catatonit cni-plugins libmnl"
+            apks="$apks libnftnl-libs iptables ip6tables libslirp slirp4netns"
+            apks="$apks shadow-uidmap fuse-common fuse3-libs fuse3 fuse-overlayfs"
+            apks="$apks containers-common libgpg-error libassuan"
+            apks="$apks ncurses-terminfo-base ncurses-libs libgcrypt libsecret"
+            apks="$apks pinentry libbz2 gmp nettle p11-kit libtasn1 libunistring"
+            apks="$apks gnutls libksba gdbm libsasl libldap npth sqlite-libs"
+            apks="$apks gnupg gpgme podman"
```

Comment:
Why do you include all the dependencies explicitly? They should be pulled into the image automatically.

Looks like the `LIMA_INSTALL_DOCKER` option also explicitly enumerates prerequisites. And that should be fixed too. Duplicating dependencies here will just become a maintenance issue as they change over time.

You can check `/media/cdrom/apks/x86_64` to see the bundled packages. I checked the `ci` edition, and all the python packages used by `cloud-init` are included, even though they are not explicitly listed here.

Reply:
Oh, that would be great!

I thought it was priming a cache or something and needed the .apk names...